### ISSUE TITLE
Improve and document ShapeWizard's classifier component :robot: 

### ISSUE THREAD

**Issue by ventus550** [WizzWriters/spelldraw-app#33]
1. Increase "other" classification bias
2. Visualize networks attention (shap, lime or via integrated gradients)
3. Refactor relevant training code
4. Employ more size efficient architecture
